.travis.yml

```diff
@@ -15,12 +15,15 @@ python:
 
 install:
   - pip install ansible
+  # - pip install shade shade-ansible
```

Comment:
since there's a requirements now, why not simplify `install` by using it.

Reply:
For the time being not everything is needed for the tests and it will speed things to not install shade that has a lot of dependencies.
To be changed once more through tests will be in place.